pkg/validation/spec/parameter.go

```diff
@@ -38,26 +39,31 @@ type ParamProps struct {
 //
 // There are five possible parameter types.
 // * Path - Used together with [Path Templating](#pathTemplating), where the parameter value is actually part
-//   of the operation's URL. This does not include the host or base path of the API. For example, in `/items/{itemId}`,
-//   the path parameter is `itemId`.
+//
```

Comment:
if possible, separate the go1.19 gofmt whitespace-only changes into their own leading commit